docs/source/gbd2017_models/risks/low_birthweight_short_gestation/index.rst

```diff
@@ -51,3 +52,162 @@ described below, and relative to the joint TMREL.
+|  328     | upper respiratory tract infections                      | 
++----------+---------------------------------------------------------+
+|  329     | otitis media                                            |
++----------+---------------------------------------------------------+
+|  333     | pneumococcal meningitis                                 |
++----------+---------------------------------------------------------+
+|  334     | H influenzae type B meningitis                          |
++----------+---------------------------------------------------------+
+|  335     | meningococcal meningitis                                |
++----------+---------------------------------------------------------+
+|  336     | other meningitis                                        |
++----------+---------------------------------------------------------+
+|  337     | encephalitis                                            |
++----------+---------------------------------------------------------+
+|  381     | neonatal preterm birth complications                    |
++----------+---------------------------------------------------------+
+|  382     | neonatal encephalopathy due to birth asphyxia and trauma|
++----------+---------------------------------------------------------+
+|  383     | neonatal sepsis and other neonatal infections           |
++----------+---------------------------------------------------------+
+|  384     | hemolytic disease and other neonatal jaundice           |
++----------+---------------------------------------------------------+
+|  385     | other neonatal disorders                                |
++----------+---------------------------------------------------------+
+|  686     | sudden infant death syndrome                            |
++----------+---------------------------------------------------------+
+
+
+Vivarium modelling strategy
++++++++++++++++++++++++++++
+
+First, we convert the GBD 500g-2weeks birthweight-ga bins/categories to a joint continuous distribution using Abie's notebook (add link?). We assume a uniform distribution within each bin/category. 
+
+.. note ::
+    That this is likely biasing towards overestimating extreme birthweights or gestational ages. For example, in the 0-500g category, most babies are probably pretty close to 500g, not equally probable to be 1 gram versus 499 grams.
+
+Because the relative risks from GBD are for all-cause mortality in the early and late neonatal period, we first define all-cause mortality rate (ACMR) as the sum of: 
+
+   - mortality from causes that are affected by lbwsg and modelled in the sim (green)
+   - mortality from causes that are affected by lbwsg but not modelled in the sim (blue)
+   - mortality from causes that are unaffected by lbwsg, modelled or not modelled (salmon)
+
+An example of these causes from the large-scale-food fortification concept model diagram is shown below: 
```

Comment:
You could add a reference to make the text an actual link to the LSFF concept model.

Reply:
where do I find that how the links are referenced? for example I know its this: 

:ref:`intermediate <_where do I find how this is called?_>`

Reply:
Here's the Sphinx documentation for how to reference different locations within the documentation: https://www.sphinx-doc.org/en/1.5/markup/inline.html#cross-referencing-arbitrary-locations

If you go to the source code for the [LSFF concept model](https://vivarium-research.readthedocs.io/en/latest/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.html), you should see this after the comments at the top:

```
.. _2017_concept_model_vivarium_conic_lsff:

=============================================
Vivarium CoNIC Large Scale Food Fortification
=============================================

Model Overview
--------------

Objective
+++++++++

Intervention Definitions
++++++++++++++++++++++++

Questions of Interest
+++++++++++++++++++++

Scope of Modeling
+++++++++++++++++


Concept Model Diagram
---------------------

.. image:: lsff_concept_model_diagram.svg
```

The first line above shows you that the reference label for the title of the document is `2017_concept_model_vivarium_conic_lsff`. Thus, to link to the concept model document, you would do :ref:`large-scale-food fortification concept model <2017_concept_model_vivarium_conic_lsff>`. (I'm not sure how to escape backticks, so this is not quite rendering correctly...)

If you wanted to link to the concept model diagram directly, you would have to edit the concept model `.rst` document and add a new reference label before the diagram, e.g.

```
.. _lsff_concept_model_diagram:

Concept Model Diagram
---------------------

.. image:: lsff_concept_model_diagram.svg
```

And then you could link to it like :ref:`large-scale-food fortification concept model diagram <lsff_concept_model_diagram>`.